en/integrations/plugin-packs/procedures/network-fortinet-fortiweb-snmp.md

```diff
@@ -0,0 +1,168 @@
+---
+id: network-fortinet-fortiweb-snmp
+title: Fortinet FortiWeb SNMP
+---
+
+## Plugin Pack Assets
+
+### Monitored Objects
+
+The Plugin Pack Fortinet FortiWeb SNMP collects metrics for:
+* Interfaces
+* Proxy
+* System
+
+### Discovery rules
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Services-->
+
+| Rule name                                 | Description                                                           |
+| :---------------------------------------- | :-------------------------------------------------------------------- |
+| Net-Fortinet-Fortiweb-SNMP-Interface-Name | Discover network interfaces and monitor bandwidth utilization         |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+### Collected Metrics
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Interfaces-->
+
+| Metric name                                            | Description                                         | Unit |
+| :----------------------------------------------------- | :-------------------------------------------------- | :--- |
+| status                                                 | Status of the interface                             |      |
+| *interface\_name*\#interface.traffic.in.bitspersecond  | Incoming traffic going through the interface        | b/s  |
+| *interface\_name*\#interface.traffic.in.percentage     | Percentage of the interface's *in* bandwidth usage  | %    |
+| *interface\_name*\#interface.traffic.out.bitspersecond | Outgoing traffic going through the interface        | b/s  |
+| *interface\_name*\#interface.traffic.out.percentage    | Percentage of the interface's *out* bandwidth usage | %    |
+
+A regexp filter is available to target a specific interface identifier - ifName [```--interface='^eth0$' --name```]
+
+<!--Proxy-->
+
+| Metric name                 | Description                   | Unit |
+| :-------------------------- | :---------------------------- | :--- |
+| proxy.connections.count     | current number of connections |      |
+| proxy.connections.persecond | average number of connections |      |
+| proxy.services.count        | number of services            |      |
```

Comment:
Please start at least the description with an uppercase even if the metric name is self-explanatory.